.gitlab-ci.yml

```diff
@@ -1,24 +1,15 @@
 
 ---
 
-#
-
 include:
-  - /.gitlab-ci/variables.yml
-  - /.gitlab-ci/extends.yml
-  #
-  - /.gitlab-ci/stages/quality.yml
-  - /.gitlab-ci/stages/register.yml
-  - /.gitlab-ci/stages/notify.yml
-  - /.gitlab-ci/stages/deploy.yml
-
-#
+  - project: SocialGouv/gitlab-ci-yml
+    file: /autodevops_simple_app.yml
+    ref: v9.0.0
 
-stages:
-- "Code Quality"
-- "Registration"
-- "Deploy"
-- "Notify Finished Deployment"
-- "Delete K8S Objects"
+variables:
+  PROJECT: work-in-france
+  RANCHER_PROJECT_ID: "c-kk8xm:p-gj8cn" # "default" project id here
+  VALUES_FILE: ./.k8s/app.values.yml # Your values
+  DOCKER_BUILD_ARGS: "--shm-size 512M"
 
-#
+...
```

Comment:
```suggestion

```

Reply:
https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

Reply:
I personally find it strange to have `...` at the end of the file.
If you argue that we have `---` at the start of the file, then just remove both.
I think that it's no necessary to add misleading optional characters in our projects.

Reply:
I  dont see any "..." or "---" in YML in GitLab official repos

Reply:
C'est enlevé comme vous le souhaitiez.